gradle-plugin/plugin/src/main/java/com/yelp/codegen/plugin/GenerateTask.kt

```diff
@@ -83,13 +92,13 @@ open class GenerateTask : DefaultTask() {
             specName ${'\t'} ${specName ?: "[ DEFAULT ] $DEFAULT_NAME"}
             specVers ${'\t'} ${specVersion ?: "[ DEFAULT ] $DEFAULT_VERSION"}
```

Comment:
`?: "[ DEFAULT ] $DEFAULT_VERSION` should go

Reply:
Same as line `109` (that I can't comment):
`params.add(specVersion ?: DEFAULT_VERSION)`

Reply:
Not sure I follow, you want to remove all the leading "[ DEFAULT ]" from the logs ? That's fine with me but I think we should do it for all params ?

Reply:
Sorry I wasn't clear, my fault.
The `specVersion` variable is always not null. So the elvis operator is not needed and the "null" branch will never run (so you can remove it).

Reply:
Indeed :+1: . IntelliJ did it for me after I pressed Alt + Enter :)

Reply:
Looking at the `convention`, thing, I simplified a bit the logs code but a side effect is that the '[ DEFAULT ]' indication is lost for all properties. Let me know if that's ok or if I should put it back.